docs/getting_started.rst

```diff
@@ -43,17 +43,44 @@ Serialization
              return value, 1
          return json.dumps(value), 2
 
-    def json_deserializer(key, value, flags):
+     def json_deserializer(key, value, flags):
         if flags == 1:
             return value
         if flags == 2:
             return json.loads(value)
         raise Exception("Unknown serialization format")
 
-    client = Client(('localhost', 11211), serializer=json_serializer,
-                    deserializer=json_deserializer)
-    client.set('key', {'a':'b', 'c':'d'})
-    result = client.get('key')
+     client = Client(('localhost', 11211), serializer=json_serializer,
+                     deserializer=json_deserializer)
+     client.set('key', {'a':'b', 'c':'d'})
+     result = client.get('key')
+
+pymemcache provides a default pickle based serializer:
```

Comment:
I would link `pickle` to the Python docs here, and also hyphenate "pickle-based".